.github/workflows/Trgr InitDev workflows in Implementation repo.yml

```diff
@@ -3,7 +3,6 @@ name: Trgr InitDev workflows in Implementation repo
 on:
   pull_request:
     branches: [main, 'b-v*-patch*','feature-*']
-  workflow_dispatch:
```

Comment:
Is there a use case that would benefit from allowing this workflow to be manually executed?

Reply:
No

Reply:
Personally, I usually like to leave `workflow_dispatch` for manual re-execution or experiment on an alternate branch.

Reply:
I added `workflow_dispatch` back to allow for manual re-executions/experimentation.